File: SWIFT/day2.swift
// Created by Ashok Varma
import Foundation

let originalPrice : Double = Double(readLine(stripNewline: true)!)!
let interestPercent : Int = Int(readLine(stripNewline: true)!)!
let tipPercent : Int = Int(readLine(stripNewline: true)!)!

let tax : Double = (Double(interestPercent) * originalPrice)/100
let tip : Double = (Double(tipPercent) * originalPrice)/100

var finalPrice : Double = originalPrice + tax + tip
let finalPriceInt = Int(round(finalPrice))

print("The final price of the meal is $\(finalPriceInt).")

File: C#/day2.cs
using System;

class Solution
{
    static void Main(String[] args)
    {
            double price = Convert.ToDouble(Console.ReadLine());
            double tip = Convert.ToInt32(Console.ReadLine());
            double tax = Convert.ToInt32(Console.ReadLine());
            int res = (int)Math.Round(price + (price * (tip / 100)) + (price * (tax / 100)));
            Console.WriteLine($"The final price of the meal is ${res}.");
    }
}
